op.c

```diff
@@ -8941,13 +8974,19 @@ Perl_utilize(pTHX_ int aver, I32 floor, OP *version, OP *idop, OP *arg)
         }
         /* otherwise they are off */
         else {
+            if(PL_prevailing_version >= SHORTVER(5, 11))
+                Perl_ck_warner_d(aTHX_ packWARN(WARN_DEPRECATED),
+                    "Downgrading a use VERSION declaration to below v5.11 is now deprecated");
```

Comment:
The "now" is redundant, and the warning should say which release it will be removed in.